resources/assets/js/components/PostList.vue

```diff
@@ -15,27 +16,25 @@
 </template>
 
 <script>
-import PostItem from './PostItem.vue';
+import PostItem from "./PostItem.vue";
 
 export default {
     components: {
-        PostItem,
+        PostItem
     },
 
     props: {
         source: {},
-        targets: {},
+        targets: {}
     },
 
     computed: {
         recent_posts() {
-            // @todo: It'd be nice if this instead just showed those published in the last two weeks
-            //
-            //      KD: check out https://github.com/xx45/dayjs for this; 2KB and you can do
-            //      isAfter(dayjs().subtract(14, 'day');)
-
-            return this.source.posts.slice(0, 5);
+            return this.source.posts.filter(
+                post =>
+                    moment(post.published_at) > moment().subtract(14, "days")
```

Comment:
Is moment better than dayjs?

Reply:
I've only had experience with momentjs, do we use dayjs often in our projects? The methods are  pretty similar.

Reply:
The comment that you deleted from Keith gives a syntax you could use to try it with dayjs and he mentioned it being very light, so I think it would be worth trying.